gm4_smelteries/data/gm4_smelteries/functions/process.mcfunction

```diff
@@ -0,0 +1,14 @@
+# smeltery processes
+# @s - smeltery
+# called by main
+
+# removal
+execute unless block ~ ~ ~ hopper run function gm4_smelteries:destroy
+
+# verify furnace location
+execute unless block ^ ^ ^1 furnace run function gm4_smelteries:find_furnace
+
+execute if entity @s[tag=gm4_bf_has_furnace] if block ^ ^ ^1 furnace[lit=true] run function gm4_smelteries:verify_furnace
+
+# particle
+execute if block ^ ^ ^1 furnace[lit=true] run particle large_smoke ^ ^.4 ^1 0 0 0 0 5 normal @a
```

Comment:
Putting the particle here gets it stuck in the furnace when theres glass or a hopper above it. Maybe put the particle above the hopper block instead?

Reply:
Fixed with https://github.com/Gamemode4Dev/GM4_Datapacks/pull/294/commits/96ed6afb4e88885a8417eb2e4f333b370d01c455